fe/src/main/java/org/apache/doris/load/routineload/RoutineLoadScheduler.java

```diff
@@ -86,18 +86,6 @@ private void process() throws UserException {
                                      .build());
                     continue;
                 }
-                int currentTotalTaskNum = routineLoadManager.getSizeOfIdToRoutineLoadTask();
```

Comment:
There are no limit about the Routine Load Job. The queue of task will continually increase till the memory of FE is going to use out.

Reply:
We have job num limit. the config is `desired_max_waiting_jobs` of FE

Reply:
Are you sure about that ?